### PR TITLE
Fix kubectl permissions and change path

### PR DESCRIPTION
**Pull request by itskingori**
~We're getting authentication errors on production ... seems `kubectl proxy` with `--www` only allows localhost.~

Just a bunch of things that we can improve.

### CODE REVIEW

**Comment by itskingori** [comment]
@zacblazic I've removed the acceptable hosts changes in [cda130e4][1] and [738a0307][2] because that's not the right fix for the issue. Just has a thought ... the reason this isn't working when deployed is because we proxy in the back-end not the front-end ... so in the client's browser any API calls are unauthenticated.

🤕 🤦‍♂️ 

[1]: https://github.com/Intellection/docker-kubernetes-visualizer/pull/3/commits/cda130e43416be1f20463497d4b3950ba1714d9d
[2]: https://github.com/Intellection/docker-kubernetes-visualizer/pull/3/commits/738a030711726118ded3a8bf8a4931dab076bc65

**Comment by itskingori** [comment]
Edited description and title.

File: Dockerfile [hunk 1]
```diff
@@ -14,9 +14,10 @@ RUN cd /srv && \
     rm "/srv/${K8S_VISUALIZER_PKG}"
 WORKDIR "/srv/kubernetes-visualizer"
 
+RUN chmod +x /usr/local/bin/kubectl
```
Comment (adrianmoisey) [inline Dockerfile]: I think this should be moved to the upstream image

Comment (itskingori) [inline Dockerfile]: Fixed in [66d6fc86][1].

[1]: https://github.com/Intellection/docker-kubernetes-visualizer/pull/3/commits/66d6fc86ff4f38cbdfb6e0f23e923e769a01827d

Comment (adrianmoisey) [inline Dockerfile]: May be just update the changelog to say that the upstream image was bumped, in order to fix the permissions of kubectl?

Comment (itskingori) [inline Dockerfile]: Sure. Addressed in [1fe4d7fb5c][1].

[1]: https://github.com/Intellection/docker-kubernetes-visualizer/pull/3/commits/1fe4d7fb5c5b63d8b429cd4ec0cf1bac9b3b5a63

**Comment by zacblazic** [comment]
> Just has a thought ... the reason this isn't working when deployed is because we proxy in the back-end not the front-end ... so in the client's browser any API calls are unauthenticated.

This makes sense now!

**Pull request merged**